packages/components/pie-notification/src/notification.scss

```diff
@@ -38,6 +38,10 @@
         --notification-direction: row;
     }
 
+    &[position='full-width'] {
+        --notification-border-radius: var(--dt-radius-00);
```

Comment:
We can update this to use an alias token in a follow-up PR 😄

Reply:
i will add it as part of this PR https://github.com/justeattakeaway/pie/pull/1435 🫰